scipy-1.0/paper.tex

```diff
@@ -1246,7 +1241,7 @@ \section{Polynomial Interpolators: Additional Details}
 \code{BSpline} objects instead of directly manipulating \code{tck}
 tuples of knots, coefficients and the spline degree.
 
-\section*{Numerical Optimization: Additional Details}
+\subsection*{Numerical Optimization: Additional Details}
```

Comment:
@rgommers suggested moving this to front, but this causes some page layout issues. Maybe send it for review looking better like this, mentioning to reviewers that it will be moved to front before submission?